third_party/BUILD

```diff
@@ -517,14 +517,36 @@ java_import(
     ],
 )
 
+UNNECESSARY_DYNAMIC_LIBRARIES = select({
+    "//src/conditions:windows": "*.so *.jnilib",
+    "//src/conditions:darwin": "*.so *.dll",
+    "//src/conditions:darwin_x86_64": "*.so *.dll",
+    "//src/conditions:linux_x86_64": "*.jnilib *.dll",
+    # The .so file is an x86 one, so we can just remove it the CPU is not x86
+    "//src/conditions:arm": "*.so *.jnilib *.dll",
+    "//src/conditions:linux_aarch64": "*.so *.jnilib *.dll",
+    # Play it safe -- better have a big binary than a slow binary
+    # zip -d does require an argument. Supply something bogus.
+    "//conditions:default": "*.bogusextension",
+})
+
+# Remove native libraries that are for a platform different from the one we are
+# building Bazel for
```

Comment:
super-nit: please add a . at the end of the comment.

Reply:
done